website/src/_includes/flink_python_pipeline_options.html

```diff
@@ -1,150 +0,0 @@
-<!--
```

Comment:
Heads-up, this file is automatically generated as part of the Flink build target `:runners:flink:1.10:generatePipelineOptionsTablePython`.